packages/bitcore-wallet-client/lib/api.js

```diff
@@ -928,72 +793,6 @@ API.prototype.isComplete = function() {
   return this.credentials && this.credentials.isComplete();
 };
 
-/**
- * Is private key currently encrypted?
- *
- * @return {Boolean}
- */
-API.prototype.isPrivKeyEncrypted = function() {
-  return this.credentials && this.credentials.isPrivKeyEncrypted();
-};
-
-/**
- * Is private key external?
- *
- * @return {Boolean}
- */
-API.prototype.isPrivKeyExternal = function() {
-  return this.credentials && this.credentials.hasExternalSource();
-};
-
-/**
- * Get external wallet source name
- *
- * @return {String}
- */
-API.prototype.getPrivKeyExternalSourceName = function() {
-  return this.credentials ? this.credentials.getExternalSourceName() : null;
-};
-
-/**
- * Returns unencrypted extended private key and mnemonics
- *
- * @param password
- */
-API.prototype.getKeys = function(password) {
```

Comment:
We used to use this function. Will there be something similar in key.js?

Reply:
.get() ?